### ISSUE TITLE
test/verify that all nivabase stations, projects and projectstations are properly synced

### ISSUE THREAD

**Issue by haakondr** [NIVANorge/odm2-postgres-api#209]
write a script that does the following:

1. extract all stations, projects and projects_stations rows from nivabase prod
2. check if all id's are present in externalidentifier tables
3. check if the row is present in samplingfeatures, projects and projectstations tables with correct values

**Comment by haakondr** [comment]
using the following queries to extract data from nivabase, dumped to json:

```
select s.STATION_ID, s.entered_date, sp.LONGITUDE, sp.LATITUDE, st.station_type
from nivadatabase.stations s
         inner join niva_geometry.SAMPLE_POINTS sp on s.GEOM_REF_ID = sp.SAMPLE_POINT_ID
        inner join nivadatabase.station_types st on s.STATION_TYPE_ID = st.STATION_TYPE_ID;
```

```
select p.project_id,
       p.project_name,
       p.project_description,
       LISTAGG(o.O_NUMBER, ',') within group ( order by p.project_id) as onumbers
from projects p
         inner join projects_o_numbers o on p.PROJECT_ID = o.PROJECT_ID
group by p.project_id, p.PROJECT_NAME, p.project_id, p.project_name, p.project_description;
```

```
select * from projects_stations;
```

**Comment by haakondr** [comment]
# findings

- we are not picking up new onumbers for existing projects. Created issue https://github.com/NIVANorge/niva-data-producers/issues/19

**Comment by haakondr** [comment]
I reran the validation script at 2022-03-15 and concluded with the following:

- onumbers seem to be picked up. I think I previously made a false assumption about the reason for missing onumbers. Maybe this was solved by #198 
- station_id=0 is still not in odm2, but I suggest we ignore this as it seems to be a dummy project
- a lot of project stations are missing in odm2, but all are linked to stations without coordinates which we exclude by design

Apart from these, all looks good from what I can tell
